docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
   golem-shard-manager:
     build:
       context: .
-      dockerfile: golem-shard-manager/Dockerfile
+      dockerfile: golem-shard-manager/docker/x86/Dockerfile
```

Comment:
not suer if is not better to use amd64 instead of x86

https://docs.docker.com/build/building/multi-platform/